src/PasswordExposedChecker.php

```diff
@@ -140,10 +170,12 @@ private function makeRequest($hash)
     }
 
     /**
+     * Convert response body to PasswordStatus constant.
+     *
      * @param string $hash
      * @param string $responseBody
      *
-     * @return string
+     * @return string (see PasswordStatus)
```

Comment:
```suggestion
     * @return string
     * @see PasswordStatus
```

Perhaps use the `@see` tag? [Source](http://docs.phpdoc.org/references/phpdoc/tags/see.html)